### PR TITLE
Fix some of the warnings in v0.5

### PR DESCRIPTION
**Pull request by dmbates**
But see issue #117

### CODE REVIEW

**Comment by dmbates** [comment]
Okay to merge this? The AppVeyor failure seems to be related to `Libdl.find_library` on Windows, v0.5-dev.

File: src/eventloop.jl [hunk 1]
```diff
@@ -1,10 +1,10 @@
 function process_events()
     ##FIXME: a dirty fix to prevent segfault right after a sigint
     if unsafe_load(cglobal((:R_interrupts_pending,libR),Cint)) == 0
-        if Sys.OS_NAME ∈ (:Darwin, :Windows)
```
Comment (randy3k) [inline src/eventloop.jl]: This line should be window and mac

File: src/eventloop.jl [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+        if Compat.is_unix()
             ccall((:R_ProcessEvents, libR), Void, ())
         end
-        if Sys.OS_NAME != :Windows
```
Comment (randy3k) [inline src/eventloop.jl]: This should be unix.

**Comment by dmbates** [comment]
I fixed the scrambled logic re platforms as caught by @randy3k

**Pull request merged**